lib/creator/index.test.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+function replaceGeneratorName(name) {
+	return name.replace(
+      /(webpack-addons)?([^:]+)(:.*)?/g, 'generator$2');
+}
+
+describe('replaceGeneratorName', () => {
+
+	it('should replace a pattern of an addon', () => {
+		const generatorName = replaceGeneratorName('webpack-addons-thefox');
+		expect(generatorName).toEqual('generator-thefox');
```

Comment:
The regex looks more complex as just and covers more edge cases. Should we test against them?